src/widget/systemtrayicon.cpp

```diff
@@ -114,6 +114,8 @@ SystemTrayIcon::SystemTrayIcon()
                                  8, image->width(), image->height(),
                                  image->bytesPerLine(), callbackFreeImage, image);
 
+        delete image;   // Not needed anymore.
```

Comment:
Why not change `QImage*` to `QImage`? Also solves the ugly `*image = …` in line 111 above.